clusterloader2/pkg/measurement/interface.go

```diff
@@ -35,6 +35,9 @@ type MeasurementConfig struct {
 	// TemplateProvider provides templated objects.
 	TemplateProvider *config.TemplateProvider
 	// Identifier identifies this instance of measurement.
+	ClusterLoaderConfig *config.ClusterLoaderConfig
```

Comment:
Something is broken with the comments - comment about identifier should be moved above identifier, comment for this field should be moved above this field.